datastore/writer/core/writer_service.py

```diff
@@ -138,15 +138,23 @@ def write_action_worker(
 
         with make_span("write action worker"):
             self.position_to_modified_models = {}
-            with self.database.get_context():
-                event = write_request.events[0]
-                fields_with_delete = copy.deepcopy(event.fields)  # type: ignore
-                fields_with_delete.update({META_DELETED: False})
-                self.database.write_model_updates_action_worker(
-                    {event.fqid: fields_with_delete}
-                )
-            self.position_to_modified_models[0] = {event.fqid: event.fields}  # type: ignore
-            self.propagate_updates_to_redis(False)
+            fqids_to_delete: List[Fqid] = []
+            if type(write_request.events[0]) == RequestDeleteEvent:
+                """delete requests will be handled all together"""
```

Comment:
I would prefer a `# delete requests will be handled ...` here.

Reply:
deleted the comment, IMO it is not needed anymore